### PR TITLE
[Icons] Added new inheritance icon

### PR DESCRIPTION
**Pull request by cyu06**
@pkwi, would you be able to assist me to add to the icons-16, icons-20 font files?

### CODE REVIEW

File: packages/icons/resources/icons/icons.json [hunk 1]
```diff
@@ -3232,5 +3232,12 @@
     "tags": "folder, mail, file, message, notifications, warning",
     "group": "file",
     "content": "\\e7d4"
+  },
+  {
+    "displayName": "Inheritance",
+    "iconName": "inheritance",
+    "tags": "arrows, derive",
+    "group": "action",
+    "content": "\\e7e0"
```
Comment (pkwi) [inline packages/icons/resources/icons/icons.json]: change to e7d5 to follow up order

**Comment by pkwi** [comment]
@cyu06 we will need to add these to the icon font. Could you include the SVG files here?

**Comment by cyu06** [comment]
@pkwi, sent you directly since I couldn't attach SVGs here.

**Comment by cyu06** [comment]
@pkwi, ping.

**Comment by pkwi** [comment]
@cyu06 I haven't received your files

**Comment by cyu06** [comment]
This is ready for another pass

**Pull request merged**